### ISSUE TITLE
Connection to Redshift

### ISSUE THREAD

**Issue by andreiconstantinescu** [postgraphql/postgraphql#458]
Hey guys :) 

I want to connect to a redshift cluster with this package. Although using `pg` on its own to connect works, if I use `postgraphql cli` and pass it the same config i get this error:

```bash
error: syntax error at or near "]"
    at Connection.parseE (/Users/<user>/.config/yarn/global/node_modules/pg/lib/connection.js:569:11)
    at Connection.parseMessage (/Users/<user>/.config/yarn/global/node_modules/pg/lib/connection.js:396:17)
    at Socket.<anonymous> (/Users/<user>/.config/yarn/global/node_modules/pg/lib/connection.js:132:22)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:191:7)
    at readableAddChunk (_stream_readable.js:178:18)
    at Socket.Readable.push (_stream_readable.js:136:10)
    at TCP.onread (net.js:560:20)
```
Any ideas how to fix it?

Thanks! :) 

LE: forgot to mention that i pass the db and schema when trying to connect

**Comment by benjie** [comment]
Hi @andreiconstantinescu, please try running PostGraphQL with the environmental variable DEBUG="postgraphql:postgres*"; it should output the SQL queries that are being ran which should give us some more hints. I wonder if it's related to parsing your database URL (e.g. funky password characters)?

**Comment by andreiconstantinescu** [comment]
Hey @benjie, I tried and I get the same output with no sql query. Apparently this error occurs before any queries are ran. 
Regarding the password, it doesn't have any unusual characters.

For what is worth, here is the whole output:
```bash
➜  ~ set DEBUG "postgraphql:postgres*";postgraphql -c postgres://<user>:<pass>@<amazon-redshift-endpoint>:<port>/<db> -s <schema>

PostGraphQL server listening on port 5000 🚀

  ‣ Connected to Postgres instance postgres://<amazon-redshift-endpoint>:<port>/<db>
  ‣ Introspected Postgres schema(s) <schema>
  ‣ GraphQL endpoint served at http://localhost:5000/graphql
  ‣ GraphiQL endpoint served at http://localhost:5000/graphiql

* * *

error: syntax error at or near "]"
    at Connection.parseE (/Users/<user>/.config/yarn/global/node_modules/pg/lib/connection.js:569:11)
    at Connection.parseMessage (/Users/<user>/.config/yarn/global/node_modules/pg/lib/connection.js:396:17)
    at Socket.<anonymous> (/Users/<user>/.config/yarn/global/node_modules/pg/lib/connection.js:132:22)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:191:7)
    at readableAddChunk (_stream_readable.js:178:18)
    at Socket.Readable.push (_stream_readable.js:136:10)
    at TCP.onread (net.js:560:20)
```

**Comment by benjie** [comment]
You may need `export` rather than `set` so that subprocesses import it; alternatively (assuming a bash/zsh shell) set the DEBUG var without statement divider:

`DEBUG="postgraphql:*" postgraphql -c postgres://... -s ...`

(Also, lets expand the range of DEBUG as above; at the moment my guess is that it does not like the introspection query.)

**Comment by andreiconstantinescu** [comment]
it doesn't make any difference, now I ran it under `bash`, hoping to get more details, but no :( 

```bash 
bash-3.2$ export DEBUG="postgraphql:*"; postgraphql -c postgres://<user>:<pass>@<amazon-redshift-endpoint>:<port>/<db> -s <schema>

PostGraphQL server listening on port 5000 🚀

  ‣ Connected to Postgres instance postgres://<amazon-redshift-endpoint>:<port>/<db>
  ‣ Introspected Postgres schema(s) <schema>
  ‣ GraphQL endpoint served at http://localhost:5000/graphql
  ‣ GraphiQL endpoint served at http://localhost:5000/graphiql

* * *

error: syntax error at or near "]"
    at Connection.parseE (/Users/<user>/.config/yarn/global/node_modules/pg/lib/connection.js:569:11)
    at Connection.parseMessage (/Users/<user>/.config/yarn/global/node_modules/pg/lib/connection.js:396:17)
    at Socket.<anonymous> (/Users/<user>/.config/yarn/global/node_modules/pg/lib/connection.js:132:22)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:191:7)
    at readableAddChunk (_stream_readable.js:178:18)
    at Socket.Readable.push (_stream_readable.js:136:10)
    at TCP.onread (net.js:560:20)
```

**Comment by brysgo** [comment]
Try `DEBUG=* postgraphql -c postgres://<user>:<pass>@<amazon-redshift-endpoint>:<port>/<db> -s <schema>`

**Comment by andreiconstantinescu** [comment]
@brysgo just tried... same output :(

**Comment by brysgo** [comment]
@andreiconstantinescu - greenplum has the same error with the introspection query, maybe it is the same thing?

**Comment by andreiconstantinescu** [comment]
@brysgo I think so
I tried on a normal Postgres db and works without any problems 👍

**Comment by benjie** [comment]
Try running the introspection query directly.

**Comment by brysgo** [comment]
I'm working on a pr to manually load and dump the introspection to see if it fixes this. Was going to socialize it once I give it a test.

**Comment by benjie** [comment]
Closed in favour of #470
